doc/process/jenkins_job_configuation_access.md

```diff
@@ -0,0 +1,90 @@
+# Introduction
+
+There are a number of cases were we would like working group members
+to be able to run and or edit jobs within jenkins. Examples include
+the CITGM jobs and the Benchmarking jobs.
+
+By default jobs can only be edited by build working group members
+and start/stopped etc. by collaborators. We do not give access to
+everbody to run/edit as it would be a potential security issue,
+both through the potential disclosure of secrets used as
+part of the builds, as well as a potential denial
+of service if jobs are launched maliciously.
+
+This document outlines the processes by which we will allow
+people outside the build working group to edit jobs as well
+as allowing people who are not collaborators run specific jobs.
+
+Unfortunately jenkins does not allow us to easily delegate the
+creation of new jobs.  As such the creation of a new job will
+need to be requested through an issue in the build repo.  Any
+build member can then create a blank job in the group for the
+working group which can then be edited by those with access
+to the jobs for that working group.
+
+## Ongoing access
+
+In the first case we want to be able to provide ongoing access to
+an individual to edit or run a job associated with a working group
+when this supports the efforts of a working group/team. For
+example, some members of the benchmarking team will be regularly
+modifying benchmarking jobs and therefore the ability to configure
+benchmarking jobs needs to be granted in an ongoing manner.
+
+In these cases the following will be used to decide if ongoing access
+can be provided:
+
+* Does the scope and size of the need justify providing access.
+* Is the invidual a collaborator? If so then access should be allowed
+  provided the first point is satisfied.
+* Length and consistency of involvement with Node.js working groups
+  and/or community.
+* Consequences to the invidudal in case of mis-bahaviour. For example,
+  would they potentially lose their job if they were reported as
+  mis-behaving to their employer ? Would being banned from involvement
+  in the Node.js community negatively affect them personally in some other
+  way ?
+* Are there collaborators who work with the individual and can vouch for
+  them.
+
+The build team will review such requests through an issue on the build
+repo which lists:
+
+* individual
+* collaborator status
+* information supporting request as per the items listed above
+* run/cancel requested: yes/no
+* edit requested: yes/no
+* Period requested for:  Indefinite or a specific from -> to
+
+The request should start with 2 lgtms from working group members and
+then the the build WG will review and vote on the request.
```

Comment:
About the build WG approval, I don't think we need a formal vote if there are no objections. I'd be ok with having the issue open for the standard 48/72 and if there are no objections then move forward. This could be longer (1 week?) if anyone would be more comfortable.

To create new jobs, if the person who is asking for a clone has permission to edit the original job, requests should be immediately granted. So, both IRC or issue are ok. Same for new blank jobs, since we can just delete everything from a clone of the same type.